ara/cli/host.py

```diff
@@ -62,19 +78,17 @@ def get_parser(self, prog_name):
         parser.add_argument(
             'host',
             metavar='<host>',
-            help='Host name or id to show',
+            help='Host id to show',
         )
         return parser
 
     def take_action(self, args):
-        try:
-            host = (models.Host.query
-                    .filter((models.Host.id == args.host) |
-                            (models.Host.name == args.host)).one())
-        except models.NoResultFound:
+        host = models.Host.query.get(args.host)
```

Comment:
Oh, that's right, one host can represent multiple host rows because it is unique per playbook id, not unique in the hosts table. Hmmm.. Let's leave it like this for the time being and see if we come up with a better UX later.